monkey/monkey_island/cc/services/config_schema/definitions/finger_classes.py

```diff
@@ -10,6 +10,7 @@
                 "SMBFinger"
             ],
             "title": "SMBFinger",
+            "safe": True,
```

Comment:
Good change, but I wonder if it wouldn't be more trivial to compare to default array. Not in defaults == not safe

Reply:
From a user perspective, "default" doesn't imply safe. There may be a case where we don't want something configured by default, but it is safe to use. From a developer perspective, the idea that "default" implies safe is tribal knowledge, which, in general, should be avoided

For code readability purposes, the intent of the code is much clearer if it's checking a property called "safe".

Code in the UI and backend can now make decisions based on the safety of a technique without also requiring access to the set of defaults.